cypress/integration/movies.spec.js

```diff
@@ -0,0 +1,28 @@
+
+import Navbar from '../page-objects/components/Navbar'
+
+describe('Movie Review', () => {
+    beforeEach(function () {
+        cy.visit('http://localhost/kino/')
+        Navbar.clickOnMovie()
+        cy.fixture('review.json').as('review')
+    })
+
+    it ('Should show a selection of all movie', function () {
+        cy.get('.book-btn').contains('Pogledaj više filmova').click()
+
+        const film = this.review['film1'] 
+        cy.get('.form-control').select(film)
+        cy.contains('Pogledaj').click()
+
+        cy.get('h5').contains('Naziv filma:').should('be.visible')

```

Comment:
This cy.get('h5') will return the list of all h5 elements on the page. Maybe you could try getting the specific element directly cy.get('Naziv filma:') and see if you can assert that directly. Or even directly cy.contains('Naziv filma:'). Try reading about how cy.get() and cy.contains() can be used in this case as the same.